http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/CromwellDao.scala

```diff
@@ -0,0 +1,10 @@
+package org.broadinstitute.dsde.workbench.leonardo.dao
+
+import cats.mtl.Ask
+import org.broadinstitute.dsde.workbench.leonardo.{AppContext, CloudContext, RuntimeName}
+
+trait CromwellDao[F[_]] {
+  def getStatus(cloudContext: CloudContext, runtimeName: RuntimeName)(implicit
```

Comment:
I think it should be called `appName` instead of `runtimeName`...runtimes in Leo are referring to either VMs or dataproc clusters specifically. Not k8s apps